zammad/README.md

```diff
@@ -32,14 +32,6 @@ See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_h
 helm show values zammad/zammad
 ```
 
-### Important note for NFS filesystems
```

Comment:
is this not true anymore?
in this version s3 and fs storage would be possible right?
so somebody could still use nfs and the info would be still helpfull?

Reply:
I believe this is not true anymore, because Zammad now requires redis to be present, and if redis is available, then the web socket server will use it by default for storage. So the issue should be solved.